docs/guides/ci-cd.md

```diff
@@ -8,34 +8,34 @@ search:
+Note that Prefect's primary ways for creating deployments, a `.deploy` flow method or a `prefect.yaml` configuration file, are both designed for building and pushing images to a Docker registry.
 
-## Getting started with GitHub Actions and Prefect
+## Get started with GitHub Actions and Prefect
 
-In this example, you'll write a GitHub Actions workflow that will run each time you push to your repository's `main` branch. This workflow will build and push a Docker image containing your flow code to Docker Hub, then deploy the flow to Prefect Cloud.
+In this example, you'll write a GitHub Actions workflow that runs each time you push to your repository's `main` branch. This workflow builds and pushes a Docker image containing your flow code to Docker Hub, then deploys the flow to Prefect Cloud.
 
 ### Repository secrets
 
 Your CI/CD process must be able to authenticate with Prefect in order to deploy flows.
 
-Deploying flows securely and non-interactively in your CI/CD process can be accomplished by saving your `PREFECT_API_URL` and `PREFECT_API_KEY` [as secrets in your repository's settings](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions) so they can be accessed in your CI/CD runner's environment without exposing them in any scripts or configuration files.
+Deploy flows securely and non-interactively in your CI/CD process by saving your `PREFECT_API_URL` and `PREFECT_API_KEY` [as secrets in your repository's settings](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions). This allows them to be accessed in your CI/CD runner's environment without exposing them in any scripts or configuration files.
 
 In this scenario, deploying flows involves building and pushing Docker images, so add `DOCKER_USERNAME` and `DOCKER_PASSWORD` as secrets to your repository as well.
 
-You can create secrets for GitHub Actions in your repository under **Settings -> Secrets and variables -> Actions -> New repository secret**:
+Create secrets for GitHub Actions in your repository under **Settings -> Secrets and variables -> Actions -> New repository secret**:
 
 ![Creating a GitHub Actions secret](/img/guides/github-secrets.png)
 
-### Writing a GitHub workflow
+### Write a GitHub workflow
 
-To deploy your flow via GitHub Actions, you'll need a workflow YAML file. GitHub will look for workflow YAML files in the `.github/workflows/` directory in the root of your repository. In their simplest form, GitHub workflow files are made up of triggers and jobs.
+To deploy your flow through GitHub Actions, you need a workflow YAML file. GitHub looks for workflow YAML files in the `.github/workflows/` directory in the root of your repository. In their simplest form, GitHub workflow files are made up of triggers and jobs.
```

Comment:
Thank you for making me face my addiction to "via" lol

Reply:
It's a great word, but not a friend to SEO or non-English speakers 😬